gtsfm/data_association/data_assoc.py

```diff
@@ -107,12 +109,19 @@ def run(
         for i, camera in cameras.items():
             triangulated_data.add_camera(i, camera)
 
+        input_tracks_classification: Optional[List[TriangulationExitCode]] = None
```

Comment:
Maybe input_tracks_exit_codes is a better name, classification is a bit unclear.

Reply:
Actually exit_codes_wrt_gt or something, to be more clear.

Reply:
Changed it. Thanks